public/auth_start.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html><html><head><script defer="defer" src="app.12e3cc787241f40172f1.js"></script></head><body><script>const params = new URLSearchParams(window.location.search);
+<!doctype html><html><head><script defer="defer" src="app.135da44884644413d02d.js"></script></head><body><script>const params = new URLSearchParams(window.location.search);
```

Comment:
You only need to commit the latest js bundle (i.e. app.135da44884644413d02d.js) with your PR. All the other versions can be deleted/reverted.

Reply:
sounds good, I deleted the other one in this PR